virttest/iscsi.py

```diff
@@ -831,9 +737,6 @@ def export_target(self):
         # Save configuration
         process.system("targetcli / saveconfig")
 
-        # Restart iSCSI service
-        restart_iscsid()
```

Comment:
How can you make sure restart_iscsid() here is not necessary?

Reply:
> How can you make sure restart_iscsid() here is not necessary?

The caller function **export_target** sets up the iscsi target, as well as the server side, while the iscsid is the initiator  , as well as the client side. Since they don't establish a connection before **export_target**, the client side iscsid doesn't need to restart. Just like you don't have to restart your browser for a new website.